common/src/state/mod.rs

```diff
@@ -585,6 +589,19 @@ impl State {
             }
             MessageEvent::RecipientRemoved { conversation } => {
                 if let Some(chat) = self.chats.all.get_mut(&conversation.id()) {
+                    // Also remove the recipient from the active call if present
```

Comment:
if we do this, we should also remove the participant from all pending calls.

also, and more importantly, this won't actually boot the person from the call, just from the UI. @stavares843 Blink would need a new feature to boot someone from a call. I think that if we really want this, we should make a separate ticket for it and add the support to Blink first.